content/blog/software-metrics-workshop-report.md

```diff
@@ -0,0 +1,284 @@
+
+**Discussion topics**
+
+We began the workshop with a group discussion on whether we should focus on software credit and incentives in general or focus specifically on machine actionable/automated ways of assessing software leading which lead to credit measures and incentives. 
+
+We focused the discussion on what an ideal credit scenario might look like and how to better reward software through citation. In particular we tried to unpack the intent of citation in the context of research software:
+
+- Do I want to cite software to **provide credit**
+
+- Do I want to cite software **for reproducibility**
+
+- Do I want to cite software to **prove my credibility and respectability** (i.e. show that I understand the area/quality signaling/ paying my dues)
+
+- Do I want to cite software to provide **transparency** of my results ("this is what I did")
+
+**Scope of the Software Institute**
+
+What should a software sustainability institute focus on with respect to software credit? Participants advocated for an institute that would support software at all levels, not just software that has been published or archived. Such an institute should support:
+
+
+
+*   New and experienced package authors
+*   Those that rely on scripts as part of their workflow.
+*   How can researchers who share code (that is not packaged in software) get credit for their work in a way that is analogous to credit for methodology?
+*   Can researchers get credit for lowering the cost of understanding and extending research (through software)?
+*   The institute could work towards helping scientists get to the next stage from simply sharing scripts. This could take the form of training and support to develop vetted, published software packages that follow community centric best practices. 
+
+    The institute could also help create metrics that would appeal to tenure committees. Software is making its way to[ tenure letters](https://cos.io/blog/are-reproducibility-and-open-science-starting-matter-tenure-and-promotion-review/) and URSSI could play a better role in placing research software in the right context for evaluation. \
```

Comment:
"would appeal to" -> "would be useful for"
"[ tenure letters]" -> " [tenure letters]"
"play a better role" -> "play a role"